src/content/translations/ja/developers/index.md

```diff
@@ -1,18 +1,30 @@
 ---
-title: デベロッパー
+title: 開発者
+meta:
```

Comment:
This is incorrect - as mentioned in the issue (#1484):
>     - Remove any `og:title` or `og:description` properties - replace those contents with `title` & `description` field names

There should no longer be a `meta` field in the frontmatter, just `title` & `description` - sorry if this wasn't clear! Always a good idea to check other translated files to see the proper format...